src/i18n/content/kr/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-view-your-data.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Overview of OpenTelemetry data in the UI
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: The New Relic UI offers a lot of options for filtering and viewing data from OpenTelemetry.
+redirects:
+  - /docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-view-your-data
+---
+
+After you send OpenTelemetry data to New Relic, you can use a variety of UI pages to analyze it. Take a look at these options:
```

Comment:
just something I'm wondering that we can talk about later maybe: was there a request to add all this info for OTel? Because it seems like we could go into a lot of these pages for other features (like APM) and if we did that, we'd have significant overlap, as these UI pages are similar across board. So I'm just wondering if you see this approach as scalable if we wanted to document these pages for APM (for example) or other features.